frontend/src/components/AddEmployeeForm.tsx

```diff
@@ -0,0 +1,125 @@
+                                        className="form-control"
+                                        value={name}
+                                        onChange={(e) => setName(e.target.value)}
+                                    >
+                                    </input>
+                                </div>
+                                <div className="from-group mb-2">
+                                    <label className="form-label">Email Adress</label>
+                                    <input
+                                        type="email"
+                                        placeholder="Enter a Email"
+                                        name="emailId"
+                                        className="form-control"
+                                        value={emailId}
+                                        onChange={(e) => setEmailId(e.target.value)}
+                                    >
+                                    </input>
+                                </div>
+                                <div className="from-group mb-2">
+                                    <label className="form-label">Position</label>
+                                    <input
+                                        type="text"
+                                        placeholder="exp. Fullstack-Developer"
+                                        name="position"
+                                        className="form-control"
+                                        value={position}
+                                        onChange={(e) => setPosition(e.target.value)}
+                                    >
+                                    </input>
+                                </div>
+
+
+                                <div className="from-group mb-2">
+                                    <label className="form-label">Gender</label>
+                                    <Form.Select placeholder="gender" name="gender" className="form-control" value={gender} aria-label="Default select example" onChange={(e) => setGender(e.target.value)}>
+                                        <option>Select</option>
+                                        <option value="Male">Male</option>
+                                        <option value="Female">Female</option>
+                                    </Form.Select>
+                                    {/*<input*/}
+                                    {/*    type="text"*/}
+                                    {/*    placeholder="gender"*/}
+                                    {/*    name="gender"*/}
+                                    {/*    className="form-control"*/}
+                                    {/*    value={gender}*/}
+                                    {/*    onChange={(e) => setGender(e.target.value)}*/}
+                                    {/*>*/}
+                                    {/*</input>*/}
+                                </div>
```

Comment:
Elvedin hat mir schonmal den Kommentar da gelassen, keine Kommentare in den Main zu mergen, also würde ich dir hier auch empfehlen, das zu entfernen wenn du es nicht mehr brauchst